### PR TITLE
Update page aa1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently the page can not be deployed.
- Currently the code quality is not up to standards.

**What is the new behavior (if this is a feature change)?**
- The page can now be deployed.
- The code quality has been improved by applying prettier.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -41,7 +37,7 @@ class App extends React.Component {
 
 export default App;
 
-/** 
+/**
  * SPDX-License-Identifier: (EUPL-1.2)
  * Copyright © 2019 Werbeagentur Christian Aichner
```
Comment (LucaAllmaier) [inline src/App.js]: missing -2020

File: src/components/molecules/Navbar/index.jsx [hunk 2]
```diff
@@ -1,97 +1,97 @@
...
-                        </MDBNavItem>
-                    </MDBNavbarNav>
-                    </MDBCollapse>
-                </MDBContainer>
-            </MDBNavbar>
-            {collapseID && overlay}
-            </div>
-        );
-    }
+    const { collapseID } = this.state;
```
Comment (LucaAllmaier) [inline src/components/molecules/Navbar/index.jsx:48]: add new line

**Review by LucaAllmaier — CHANGES_REQUESTED** [review]
Looks good after some changes.

File: src/components/molecules/Navbar/index.jsx [hunk 3]
```diff
@@ -1,97 +1,98 @@
...
-                    </MDBNavbarNav>
-                    </MDBCollapse>
-                </MDBContainer>
-            </MDBNavbar>
-            {collapseID && overlay}
-            </div>
-        );
-    }
+		const { collapseID } = this.state;
+
```
Comment (LucaAllmaier) [inline src/components/molecules/Navbar/index.jsx:49]: indentation bug again?

**Review by LucaAllmaier — APPROVED** [review]
Looks good.

File: src/components/molecules/Navbar/index.jsx (hunk 3 above)
Comment (Aichnerc) [reply src/components/molecules/Navbar/index.jsx:49]: Thanks for pointing that out.

**Pull request merged**